test/components/categoryAxisTests.ts

```diff
@@ -108,4 +108,19 @@ describe("Category Axes", () => {
 
     svg.remove();
   });
+
+  it("axis should request more space if there's not enough space to fit the text", () => {
+    var svg = generateSVG(300, 300);
+    var years = ["2000", "2001", "2002", "2003"];
+    var scale = new Plottable.Scale.Ordinal().domain(years);
+    var axis = new Plottable.Axis.Category(scale, "bottom");
+    axis.renderTo(svg);
+    var requestedSpace = axis._requestedSpace(300, 10);
+    assert.isTrue(requestedSpace.wantsHeight, "axis should ask for more space");
+    axis.orient("left");
+    requestedSpace = axis._requestedSpace(10, 300);
+    assert.isTrue(requestedSpace.wantsWidth, "axis should ask for more space");
```

Comment:
Can we distinguish between the two asserts in the message? "axis should ask for more space (vertical/horizontal)"